tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -210,7 +210,9 @@ public void processOsm(SourceFeature sf, FeatureCollector features) {
           maxZoom = 15.0f;
           kindRank = 2;
           if (population == 0) {
+            minZoom = 8.0f;
             if (place.equals("town")) {
+              minZoom = 9.0f;
```

Comment:
This is a Tilezen port where places with population are preferred over places without population.